tests/event_config.at

```diff
@@ -58,3 +58,130 @@ TS_MAIN
 }
 TS_RETURN_MAIN
 ]])
+
+## ------------------------ ##
+## get_options_with_err_msg ##
+## ------------------------ ##
+
+AT_TESTFUN([get_options_with_err_msg],
+[[
```

Comment:
If you are not sure what I have in mind, please have a look at:
https://github.com/abrt/libreport/blob/master/tests/local.at#L22

The test case is compiled as a local file which is a concatenation of:
```
#line __line__ "__file__"
```
and the test contents.

So, if there is a warning, an error, or  a crash the source file is `event_config.at` with correct line number and the temporary compilation file.

https://gcc.gnu.org/onlinedocs/cpp/Line-Control.html